PubNub/Data/Service Objects/PNResult.m

```diff
@@ -67,7 +67,7 @@ - (instancetype)initForOperation:(PNOperationType)operation
             processedData = [dataForUpdate copy];
             _statusCode = (([statusCode integerValue] > 200) ? [statusCode integerValue] : _statusCode);
         }
-        _serviceData = [processedData copy];
+        [self updateData:[processedData copy]];
```

Comment:
Shouldn't the copy be removed from here since updateData already creates a copy?